### PR TITLE
feat: added admin/users route

### LINKED ISSUE #158: Add admin/users page (UI only)
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend
 
image
Located at `admin/users`

Please go through the details of #153 before starting to work on this.

### PR DESCRIPTION
**Pull request by utkarshnath**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
Added added admin/users route

screen shot 2017-06-07 at 5 47 29 pm

Fixes #158
https://young-wildwood-91762.herokuapp.com/admin/users
@niranjan94 @geekyd plz review

### CODE REVIEW

File: app/templates/admin/users.hbs [hunk 1]
```diff
@@ -1 +1,101 @@
-{{outlet}}
+<div class="ui grid stackable">
+  <div class="row">
+    <div class="twelve wide column">
+      <div class="ui basic buttons">
+        <button class="ui button">{{t 'All Users'}}</button>
+        <button class="ui button">{{t 'Active Users'}}</button>
+        <button class="ui button">{{t 'Deleted Users'}}</button>
+      </div>
+      {{#ui-dropdown class='labeled icon button'}}
```
Comment (niranjan94) [inline app/templates/admin/users.hbs]: Remove this. Will be handled by ember-model-table

File: app/templates/admin/users.hbs [hunk 2]
```diff
@@ -1 +1,101 @@
...
+        <div class="default text">{{t 'No. of entries'}}</div>
+        <div class="menu">
+          <div class="item">{{t '10'}}</div>
+          <div class="item">{{t '25'}}</div>
+          <div class="item">{{t '50'}}</div>
+          <div class="item">{{t '100'}}</div>
+        </div>
+      {{/ui-dropdown}}
+    </div>
+    <div class="four wide column right aligned">
```
Comment (niranjan94) [inline app/templates/admin/users.hbs]: Remove this. Will be handled by ember-model-table

Comment (utkarshnath) [inline app/templates/admin/users.hbs]: @niranjan94 done

**Comment by utkarshnath** [comment]
@niranjan94 @CosmicCoder96 plz review

File: app/routes/admin/users.js [hunk 3]
```diff
@@ -3,4 +3,59 @@ import Ember from 'ember';
...
+  titleToken() {
+    return this.i18n.t('Users');
+  },
+  model() {
+    return [{
+      name        : 'Testname 1',
+      email       : 'Testname1@gmail.com',
+      status      : 'active',
+      systemRoles : 'unverified user',
+      eventRoles  : 'role1',
```
Comment (niranjan94) [inline app/routes/admin/users.js]: Should be an array like 
```js
[
   {
       name: 'Organizer',
       event: {
           name: 'Event One'
       }
   }
]
```

File: app/routes/admin/users.js [hunk 4]
```diff
@@ -3,4 +3,59 @@ import Ember from 'ember';
...
 export default Route.extend({
+  titleToken() {
+    return this.i18n.t('Users');
+  },
+  model() {
+    return [{
+      name        : 'Testname 1',
+      email       : 'Testname1@gmail.com',
+      status      : 'active',
+      systemRoles : 'unverified user',
```
Comment (niranjan94) [inline app/routes/admin/users.js]: Should be an array

File: app/routes/admin/users.js [hunk 5]
```diff
@@ -3,4 +3,59 @@ import Ember from 'ember';
...
+      eventRoles  : 'role1',
+      userLinks   : ['Sessions', 'Events', 'Tickets', 'Settings'],
```
Comment (niranjan94) [inline app/routes/admin/users.js]: These user links are constant for all users. Please actually try out the current system to see how things are.

File: app/templates/admin/users.hbs [hunk 6]
```diff
@@ -1 +1,83 @@
...
+              </td>
+              <td>
+                <div>
+                  {{user.eventRoles}}
+                </div>
+              </td>
+              <td>
+                <div class="ui ordered list">
+                  {{#each user.userLinks as |session|}}
+                    <div class="item">{{session}}</div>
```
Comment (niranjan94) [inline app/templates/admin/users.hbs]: Each one is a link

**Comment by utkarshnath** [comment]
@niranjan94 updated plz check
https://young-wildwood-91762.herokuapp.com/admin/users

File: app/routes/admin/users.js [hunk 7]
```diff
@@ -3,4 +3,95 @@ import Ember from 'ember';
...
+        },
+        {
+          name  : 'Organizer',
+          event : {
+            name: 'Event Two'
+          }
+        }
+      ],
+      createdAt  : new Date(),
+      lastAccess : new Date()
```
Comment (niranjan94) [inline app/routes/admin/users.js]: rename to `lastAccessedAt`

**Comment by utkarshnath** [comment]
@niranjan94 updated

File: app/routes/admin/users.js [hunk 8]
```diff
@@ -3,4 +3,95 @@ import Ember from 'ember';
...
+        },
+        {
+          name  : 'Organizer',
+          event : {
+            name: 'Event Two'
+          }
+        }
+      ],
+      createdAt    : new Date(),
+      lastAccessAt : new Date()
```
Comment (niranjan94) [inline app/routes/admin/users.js]: `lastAccessedAt`

Comment (utkarshnath) [inline app/routes/admin/users.js]: @niranjan94 sorry I missed that, updated

**Pull request merged**